### PR TITLE
add 1.18 support

### PR DESCRIPTION
**Pull request by andrei1058**
- add 1.18 support
- sideboard update (1.17 and 1.18)

### CODE REVIEW

**Review by iiProCraft — APPROVED** [review]
LGTM

**Comment by ajgeiss0702** [comment]
We should probably fix the ci issue before merging

File: bedwars-plugin/pom.xml [hunk 1]
```diff
@@ -230,6 +238,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
+                <version>3.3.0-SNAPSHOT</version>
```
Comment (ajgeiss0702) [inline bedwars-plugin/pom.xml]: Heres why its not compiling

It doesnt seem like there is `3.3.0-SNAPSHOT`. It looks like the latest is `3.2.4`

Comment (TeksuSiK) [reply bedwars-plugin/pom.xml]: Plugin repository URL is missing. Default Maven plugin repository dosen't contain snapshot versions.

```xml
<pluginRepositories>
  <pluginRepository>
    <id>apache.snapshots</id>
    <url>https://repository.apache.org/snapshots/</url>
  </pluginRepository>
</pluginRepositories>
```

Comment (andrei1058) [reply bedwars-plugin/pom.xml]: It's a snapshot!

**Comment by TeksuSiK** [comment]
Bro, just do that I have written in review. You don't have to add this to the `repositories` section, but to the `pluginRepositories` section

**Comment by andrei1058** [comment]
> `pluginRepositories`

I don't know why, but your review is not shown.

**Comment by ghost** [comment]
This doesn't appear to fix #101. Still having the same issues as described there.

**Pull request merged**